ocp_install_env.sh

```diff
@@ -98,16 +98,14 @@ EOF
 }
 
 function additional_trust_bundle() {
-    if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ] && [ -z $MIRROR_IMAGES ]; then
-cat <<EOF
-additionalTrustBundle: |
-$(echo ${ADDITIONAL_TRUST_BUNDLE} | awk '{ print " ", $0 }')
-EOF
-    elif [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ] && [ ! -z "$MIRROR_IMAGES" ]; then
-cat <<EOF
-$(echo ${ADDITIONAL_TRUST_BUNDLE} | awk '{ print " ", $0 }')
-EOF
+  if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ]; then
+    if [ -z $MIRROR_IMAGES ]; then
+      echo "additionalTrustBundle: |"
     fi
+cat << EOF
```

Comment:
This cat is unnecessary, it makes this really hard to read for no reason.

Actually the other one isn't necessary either. All you need is:

```
awk '{ print " ", $0 }' "${ADDITIONAL_TRUST_BUNDLE}"

Reply:
Done